packages/pds-ember/app/styles/pds/theme/_light.scss

```diff
@@ -1,277 +1,79 @@
 $_module: "PDS.Theme.Light";
-// config may not be the final "base" of a theme,
-// but it'll likely be a big part of it
-@use "sass:map";
 @use "../tokens/color";
-@use "../tokens/font";
-@use "../tokens/resolutions";
 @use "../tokens/scale";
 
-// TODO: deprecate/rename vars to fit EPM naming format
-
-// TODO: update with generic set of baseline
-// variables in order to easily apply a theme.
-// (see p39-42 in October 2019 Brand Guide)
-
-// Ideally, configuration can be done via CSS custom properties
-// in order to enable simpler client-side a11y configuration.
-// This requires some auditing and research to identify which
-// settings should be client-side configurable.
-//
-// For now, we define everything in Sass.
-// Sass variables define the theme module "exports"
-// for `@use "pds/theme";` within consuming apps as `theme.*` vars.
-//
-// This should allow for the replacement, or injection of
-// new/custom themes into the module @use chain.
-
-///----------------------------------------///
-/// CORE DEFAULTS
-///----------------------------------------///
-$rem: 16px;
-$backgroundColor: color.$white;
-$color: color.$black;
```

Comment:
* Converted to reference CSS Custom Properties.
* Definition migrated to PDS.Theme.Light@styles